conjur-env/main.go

```diff
@@ -1,15 +1,16 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
-	"sync"
-	"strconv"
-	"github.com/cyberark/summon/secretsyml"
 	"github.com/cyberark/conjur-api-go/conjurapi"
+	"github.com/cyberark/summon/secretsyml"
+	"io/ioutil"
 	"os"
+	"strconv"
 	"strings"
-	"io/ioutil"
-	"encoding/json"
+	"sync"
+	"errors"
```

Comment:
you should split out the third party modules from the built in library list for easier identification (you can see an example [here](https://github.com/cyberark/secretless-broker/blob/master/pkg/secretless/plugin/sharedobj/available_plugins_test.go#L3))

Reply:
Done, thanks!

Mind taking another look?